pocs/utils/rs232.py

```diff
@@ -141,7 +141,7 @@ def write(self, value):
         if self.is_threaded:
             response = self._serial_io.write(value)
         else:
-            response = self.ser.write(value.encode())
+            response = self.ser.write("{}\n".format(value).encode())
```

Comment:
I found it confusing that SerialData was dealing with anything except bytes. How would you feel about changing the API for write (in a future PR) so that we have writebytes and writeline? The former takes bytes (no encode call), and the latter takes a text string (ASCII only?), and must call encode (and maybe append a line terminator).

Or, perhaps replace (in time) SerialData with SerialIO, conforming to the io package's API for streams... which is really what PySerial already does.